lib/helpers/preserve_data.rb

```diff
@@ -0,0 +1,11 @@
+def save_data(data, file)
+  json = JSON.generate(data)
+  File.write("./lib/preserve_data/#{file}.json", json)
+end
+
+def recover_data(file)
+  return unless File.exist?("./lib/preserve_data/#{file}.json") == true
```

Comment:
Kindly change the dir folder, it should be ```./data/(#{file}.json```, which is the folder we have on develop branch to store json files

Reply:
@Trictonicmp Done, I changed it 👍🏽